tests/river/factories.py

```diff
@@ -19,6 +19,13 @@ class Meta:
     id = factory.Sequence(lambda n: f"batch_id_{n:04d}")
     mappings = factory.LazyAttribute(lambda x: mimic_mapping())
 
+    @factory.post_generation
+    def resources(self, create, extracted, **kwargs):
```

Comment:
ok, please add a comment to link to https://factoryboy.readthedocs.io/en/stable/recipes.html#simple-many-to-many-relationship